packages/node-modules-inspector/src/app/components/display/PackageSpec.vue

```diff
@@ -21,18 +21,23 @@ const deprecation = computed(() => getDeprecatedInfo(props.pkg))
         i-catppuccin-folder-packages-open icon-catppuccin mr2 absolute left-0 top-0
       />
     </span>
-    <DisplayPackageName
-      v-tooltip="deprecation?.latest ? `Package is deprecated: ${deprecation.latest}` : undefined"
-      :name="props.pkg.name"
+    <span>
+      <DisplayPackageName
+        v-tooltip="deprecation?.latest ? `Package is deprecated: ${deprecation.latest}` : undefined"
+        :name="props.pkg.name"
+        :pkg="props.pkg"
+        :class="!deprecation?.latest ? undefined : deprecation?.type === 'future' ? 'text-orange line-through' : 'text-red line-through'"
+      />
+      <DisplayVersion
+        v-tooltip="deprecation?.current ? `Current version is deprecated: ${deprecation.current}` : undefined"
+        op-fade
+        :version="props.pkg.version"
+        prefix="@"
+        :class="{ 'text-red line-through': deprecation?.current }"
+      />
+    </span>
+    <DisplayProvenanceBadge
```

Comment:
I don't think this should be part of the `PackageSpec` component as it seems would make this component carry too much responsibility. We should do it outside.